core/iwasm/include/wasm_export.h

```diff
@@ -264,13 +264,25 @@ typedef struct LoadArgs {
 } LoadArgs;
 #endif /* LOAD_ARGS_OPTION_DEFINED */
 
+struct WasmExternInstance {
+    const char *module_name;
+    const char *field_name;
+    wasm_import_export_kind_t kind;
+    union {
+        wasm_memory_inst_t memory;
+    } u;
+};
```

Comment:
how about `typedef struct WASMExternInstance { ... } WASMExternInstance;` and using `WASMExternInstance` in other places instead of `struct WASMExternInstance`? We usually define and use structure like that way.